README.md

```diff
@@ -1,4 +1,4 @@
-# Duke project template
+# Omnitravel
```

Comment:
I like this diagram a lot, very clear to see which classes implement and use which other classes. I like that you made this the first diagram in your TP as well, great work!